ui/src/dashboards/components/dashboard_index/DashboardsIndex.tsx

```diff
@@ -8,7 +8,6 @@ import DashboardsIndexContents from 'src/dashboards/components/dashboard_index/D
 import {Page} from 'src/pageLayout'
 import SearchWidget from 'src/shared/components/search_widget/SearchWidget'
 import AddResourceDropdown from 'src/shared/components/AddResourceDropdown'
-import ImportDashboardOverlay from 'src/dashboards/components/ImportDashboardOverlay'
```

Comment:
looks like theres both a ImportDashboardOverlay and DashboardImportOverlay. should one of these be deleted?